Cargo.toml

```diff
@@ -18,36 +18,68 @@ keywords = ["scripting", "scripting-engine", "scripting-language", "embedded"]
 categories = ["no-std", "embedded", "wasm", "parser-implementations"]
 
 [dependencies]
-smallvec = { version = "1.7.0", default-features = false, features = ["union", "const_new", "const_generics"] }
```

Comment:
See if you can avoid generating many changes due to auto-reformatting. This distracts from the actual changes in the file.